lib/SIL/SILValue.cpp

```diff
@@ -729,6 +729,9 @@ CONSTANT_OWNERSHIP_BUILTIN(Trivial, Assign)
 CONSTANT_OWNERSHIP_BUILTIN(Trivial, Init)
 CONSTANT_OWNERSHIP_BUILTIN(Trivial, AtomicStore)
 CONSTANT_OWNERSHIP_BUILTIN(Trivial, Once)
+
+CONSTANT_OWNERSHIP_BUILTIN(Trivial, TSanInoutAccess)
```

Comment:
I expect to eventually have additional builtins for other kinds of TSan instrumentation (actual reads and writes) and thought I would group those together. It is premature at this point, though.